sdk/cosmos/azure_data_cosmos/src/clients/container_client.rs

```diff
@@ -208,24 +209,59 @@ impl ContainerClient {
     ///     product_name: "Product #1".to_string(),
     /// };
     /// # let container_client: ContainerClient = panic!("this is a non-running example");
-    /// let created_item = container_client
+    /// container_client
     ///     .create_item("category1", p, None)
-    ///     .await.unwrap()
-    ///     .into_body()
-    ///     .await.unwrap()
-    ///     .unwrap();
-    /// println!("Created: {:#?}", created_item);
+    ///     .await?;
+    /// # }
+    /// ```
+    ///
+    /// # Content Response on Write
+    ///
+    /// By default, the newly created item is *not* returned in the HTTP response.
+    /// If you want the new item to be returned, set the [`ItemOptions::enable_content_response_on_write`] option to `true`.
+    /// You can deserialize the returned item using [`Response::into_json_body`], like this:
+    ///
+    /// ```rust,no_run
+    /// # use azure_data_cosmos::{clients::ContainerClient, ItemOptions};
+    /// # use serde::{Deserialize, Serialize};
```

Comment:
Any reason *not* to show these? Rust examples typically do show imports.

Reply:
No particular reason! Happy to align with how other examples work.